lib/transaction/transaction.js

```diff
@@ -447,6 +447,8 @@ Transaction.prototype._fromNonP2SH = function(utxo) {
 };
 
 Transaction.prototype._fromMultisigUtxo = function(utxo, pubkeys, threshold) {
+  $.checkArgument(threshold <= pubkeys.length,
```

Comment:
Shouldn't this be "Threshold must be less than or equal to the number of publickeys"? If there are 5 pubkeys, the threshold should be 5 and lower, not greater (e.g. 1-of-5 signatures required or 5-of-5) This check may need to also be at: https://github.com/bitpay/bitcore/blob/master/lib/address.js#L278